.github/workflows/ci.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+on:
+  - push
+  - pull_request
+
+jobs:
+  build:
+    runs-on: windows-latest
+    env:
+      GOPROXY: off
+      GOFLAGS: -mod=vendor
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '^1.15.0'
+      # Run linter
+      - run: Invoke-WebRequest "https://github.com/alecthomas/gometalinter/releases/download/v2.0.12/gometalinter-2.0.12-windows-amd64.zip" -OutFile "gometalinter.zip"
```

Comment:
I think we were hoping to eventually use https://github.com/golangci/golangci-lint-action when we moved to gh actions. See https://github.com/containerd/containerd/pull/4609 for an example from containerd/containerd upstream.

Reply:
I'd like to claim that your request is out of scope of this PR. Switching to a different linter might introduce massive code diff and should better be done as a separate "change linter used by the project" step.

Reply:
The linter switch was pretty simple for AppVeyor (#935) since the only _currently enabled_ linter here is `gofmt`, but I'd agree that it can be done _after_ the GH Actions are ready, since we'd want to use the GitHub Action for it, rather than a workflow script step. And possible enable more linters.